pr_agent/tools/pr_reviewer.py

```diff
@@ -103,4 +108,37 @@ def _prepare_pr_review(self) -> str:
 
         if settings.config.verbosity_level >= 2:
             logging.info(f"Markdown response:\n{markdown_text}")
-        return markdown_text
\ No newline at end of file
+        return markdown_text
+
+    def _publish_inline_code_comments(self):
+        if settings.config.git_provider != 'github': # inline comments are currently only supported for github
+            return
+
+        review = self.prediction.strip()
+        try:
+            data = json.loads(review)
+        except json.decoder.JSONDecodeError:
+            data = try_fix_json(review)
```

Comment:
**Suggestion:** Consider handling the case where `json.loads(review)` fails but `try_fix_json(review)` also fails. This will prevent potential unhandled exceptions.
```suggestion
        try:
            data = json.loads(review)
        except json.decoder.JSONDecodeError:
            try:
                data = try_fix_json(review)
            except Exception as e:
                logging.error(f'Failed to parse review: {e}')
                return
```